.github/workflows/libxrpl.yml

```diff
@@ -0,0 +1,85 @@
+name: Upload libXRPL to Conan
+env:
+  CONAN_URL: http://18.143.149.228:8081/artifactory/api/conan/conan-non-prod
+  CONAN_LOGIN_USERNAME_RIPPLE: ${{ secrets.CONAN_USERNAME }}
+  CONAN_PASSWORD_RIPPLE: ${{ secrets.CONAN_TOKEN }}
+on:
+  pull_request:
+    paths:
+      - 'src/ripple/protocol/impl/BuildInfo.cpp'
+      - '.github/workflows/libxrpl.yml'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish:
+    name: Publish libXRPL
+    outputs:
+      outcome: ${{ steps.upload.outputs.outcome }}
+      version: ${{ steps.version.outputs.version }}
+    runs-on: [self-hosted, heavy]
+    container: rippleci/rippled-build-ubuntu:aaf5e3e
+    steps:
+      - name: Wait for essential checks to succeed
+        uses: lewagon/wait-on-check-action@v1.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          running-workflow-name: wait-for-check-regexp
+          check-regexp: '(dependencies|test).*linux.*' # Ignore windows and mac tests but make sure linux passes
```

Comment:
You could probably simplify this to just check `test`, since `test` depends on `dependencies`.

Reply:
Tried but it fails. I already forgot why i had to do it in the first place: this workflow launches together with all others. Dependencies launch before and test targets so the check-regexp above need to contain `dependencies`, otherwise it will immediately exit stating that `test` was never ran against this ref.

Reply:
Oooooh. Interesting. Thanks for the explanation.